LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
-
-Copyright (c) 2022 avimak
```

Comment:
@janek26 why dropping the LICENSE? it's useful + a standard,
you are welcome to add your handle here (i.e. `Copyright (c) 2022 avimak, janek26 and contributors`)

Reply:
added. Wasn't intentional